src/finitio/parser.pegjs

```diff
@@ -105,6 +107,11 @@ unnamed_constraint =
     return { native: e.trim() };
   }
 
+type_parameters =
+  "(" head:var_name tail:(opt_comma spacing var_name)* ")" {
```

Comment:
Let add a new `parameter_name` rule instead or reusing `var_name`.

Any strong opinion about uppercase/lowercase? I would expect `S`, `T`, `U` instead of `x`, `y`, `z`, but it probably depends whether we want Finitio to be java-ish or Haskell-ish...